src/Console/CommandCollection.php

```diff
@@ -67,7 +67,7 @@ public function add(string $name, $command)
             $class = is_string($command) ? $command : get_class($command);
             throw new InvalidArgumentException(sprintf(
                 "Cannot use '%s' for command '%s'. " .
-                "It is not a subclass of Cake\Console\Shell or Cake\Command\CommandInterface.",
+                "It is not a subclass of Cake\Console\Shell, nor does it implement Cake\Console\CommandInterface.",
```

Comment:
While this is technically correct, from a practical POV stating that it's not a subclass of `Cake\Command\Command` would make it more obvious for the user regarding how to remedy the situation.

Reply:
`Cake\Command\Command` definitely sounds like the more helpful term to use here.

Reply:
@ADmad how about this instead:

`"It is not a subclass of Cake\Console\Shell or Cake\Console\CommandInterface."`

Reply:
That's pretty much what I already suggested in follow up comment :).